src/laplace.jl

```diff
@@ -0,0 +1,332 @@
+# workaround for https://github.com/JuliaDiff/ChainRulesCore.jl/issues/470 to avoid Zygote dependency
+ignore_ad(closure) = closure()
+@non_differentiable ignore_ad(closure)
+
+struct LaplaceCache{
```

Comment:
Could you add a comment on what this should represent?
Give a super small description for each field